tests/datasource/fluent/test_spark_filesystem_datasource.py

```diff
@@ -983,6 +987,42 @@ class TimeRange:
             assert metadata[key2] == range2
 
 
+@pytest.mark.unit
```

Comment:
These unit tests are fast; having them for both `Pandas` and `Spark` eliminates any confusion about `batch_slice` support in `asset. build_batch_request()` method for file-path type `DataAsset` extensions.